science/nco/Portfile

```diff
@@ -95,10 +92,7 @@ variant mpich description {enable MPI with mpich (currently MPI is not supported
 variant openmpi description {enable MPI with openmpi (currently MPI is not supported)} {
 }
 
-variant esmf description {use ESMF (Earth System Modeling Framework)} {
-    configure.args-delete   --disable-esmf
-    configure.args-append   --enable-esmf
-    depends_lib-append      port:esmf
+variant esmf description {use ESMF (Earth System Modeling Framework) (is no longer supported)} {
```

Comment:
I don't think it will fail and probably just install it without the variant, but admittedly I am not sure. We can merge it as is for now, but then eventually you should just remove the redundant variants (perhaps after 1 year as we typically do when making a port obsolete).

Reply:
Well noted. I'll take it away in about a year from now.

Reply:
A reason to keep a compatibility variant for one year is if, for example, a variant name has changed. The compatibility variant exists to ensure that users who had the port installed with the old variant name get upgraded to the new variant name and thus don't lose any functionality. That reason doesn't apply to these three variants which truly do nothing. Variants that do nothing are only harmful and should be removed immediately. I removed them in 96c749b9c03c0e69dcdb57369b78612bd5bc079b.

Reply:
Thank you, @ryandesign, for your input and view on this. I fully concur. I guess @reneeotten and I were just a bit less intrepid. It is very helpful to hear the prevailing view on these kind of things. If I had known, I would have kicked out all three stale options right away.

Reply:
well... I pretty much told you to remove it but you didn't want to 😉